src/content/faq/General.md

```diff
@@ -9,70 +9,73 @@ questions: Why should I use GraphQL?,Is GraphQL a database language like SQL?,Do
 
 ## Why should I use GraphQL?
 
-<!-- TODO -->
+It depends on your use case, but in general, GraphQL has a few key features that stand out. For example, GraphQL enables you to:
+
+* Improve performance and stability by [sending queries](/learn/queries/) for the exact data you need. 
+* Aggregate data from [multiple UI clients](/learn/queries/#fragments) in [a single request](/learn/best-practices/#server-side-batching-caching ).
+* Create a representation of your data that feels familiar and natural ([a graph](/learn/thinking-in-graphs/#it-s-graphs-all-the-way-down-https-en-wikipedia-org-wiki-turtles-all-the-way-down)).
+* Ensure that all of your data is [statically typed](/learn/schema/) and these types inform [what queries the schema supports](/learn/introspection/).
+* Examine queries and responses through community tools like [GraphiQL](https://github.com/graphql/graphiql).
```

Comment:
We can add a point about schema evolution and that GraphQL not only has a built-in mechanism for deprecations but also greatly reduces the number of situations where you need to do breaking change.
Moreover, GraphQL actually gives you insights into what data is actually used by your clients so even in situations where you need to do breaking change you can effectively communicate it with affected clients.

Reply:
Tried to squeeze that all into a bullet point, will be included in the next commit - curious to hear what you think!